server/plugin/webhook.go

```diff
@@ -908,6 +908,19 @@ func (p *Plugin) postPullRequestReviewCommentEvent(event *github.PullRequestRevi
 		return
 	}
 
+	post := &model.Post{
```

Comment:
Putting the post above the loop reintroduces the bug that https://github.com/mattermost/mattermost-plugin-github/pull/755 solves. Can we revert the changes here that are not related to `approved` etc.?